src/include/duckdb/planner/logical_operator.hpp

```diff
@@ -37,7 +44,10 @@ class LogicalOperator {
 	//! The types returned by this logical operator. Set by calling LogicalOperator::ResolveTypes.
 	vector<LogicalType> types;
 	//! Estimated Cardinality
-	idx_t estimated_cardinality = 0;
+	idx_t estimated_cardinality;
```

Comment:
Could we turn this into a struct or class, instead of 3 separate attributes?

Reply:
The `estimated_cardinality` in the logical operator is pretty tightly coupled to a lot of `physicaloperator` constructors. For example the PhysicalTableScan has estimated_cardinality in the constructor, which is usually copied from the logical_operator. I agree changing it into a struct or class would be better, but I don't think changing every instantiation of a physical operator should be in this pull request 
```
PhysicalColumnDataScan(vector<LogicalType> types, PhysicalOperatorType op_type, idx_t estimated_cardinality)
	    : PhysicalOperator(op_type, move(types), estimated_cardinality), collection(nullptr) {
	}
```